assets/css/style.css

```diff
@@ -195,64 +228,46 @@ a {
     padding: 0.1% 0 0.1% 1%;
     text-align: left;
     position: relative;
+    z-index: 1000;
     top: 70%;
 }
 
+.carousel-inner {
+    height: 45vh;
+    /* width: 65vw; */
+}
+
 /* style for text inside the project title box */
 .project-title a {
     color: var(--space-cadet);
     font-size: smaller;
 }
 
-/* mini project week 1 */
+
+/* The Boring App */
 #project-1 {
-    background-image: url(./images/project-1-mini-project.jpeg);
-    width: 70%;
-    height: 25vh;
-    background-size: cover;
-    background-position: center;
-    flex: 1 1 70%;
+    background-position: top;
+    flex: 1 1 65vw;
 }
 
-/* who app */
+/* Weather Dashboard */
 #project-2 {
```

Comment:
consider placing these styles in a class and applying that to all three projects in place of using ids

Reply:
thanks Dom